packages/cli/src/Generate.ts

```diff
@@ -138,7 +138,7 @@ ${chalk.bold('Examples')}
         printDownloadProgress: !watchMode,
         version: enginesVersion,
         cliVersion: pkg.version,
-        dataProxy: args['--data-proxy'],
+        dataProxy: !!args['--data-proxy'],
```

Comment:
All the changes here are about not passing `undefined` values anymore (after ts-jest fixes).